web/libs/editor/src/components/ImageView/ImageView.jsx

```diff
@@ -66,37 +73,51 @@ const Region = memo(({ region, showSelected = false }) => {
   return useObserver(() => Tree.renderItem(region, region.annotation, true));
 });
 
-const RegionsLayer = memo(({ regions, name, useLayers, showSelected = false }) => {
+const RegionsLayer = memo(({ regions, name, useLayers, showSelected = false, smoothing = true }) => {
   const content = regions.map((el) => <Region key={`region-${el.id}`} region={el} showSelected={showSelected} />);
 
-  return useLayers === false ? content : <Layer name={name}>{content}</Layer>;
-});
-
-const Regions = memo(({ regions, useLayers = true, chunkSize = 15, suggestion = false, showSelected = false }) => {
-  return (
-    <ImageViewProvider value={{ suggestion }}>
-      {(chunkSize ? chunks(regions, chunkSize) : regions).map((chunk, i) => (
-        <RegionsLayer
-          key={`chunk-${i}`}
-          name={`chunk-${i}`}
-          regions={chunk}
-          useLayers={useLayers}
-          showSelected={showSelected}
-        />
-      ))}
-    </ImageViewProvider>
+  return useLayers === false ? (
+    content
+  ) : (
+    <Layer name={name} imageSmoothingEnabled={smoothing}>
+      {content}
+    </Layer>
   );
 });
 
+const Regions = memo(
+  ({ regions, useLayers = true, chunkSize = 15, suggestion = false, showSelected = false, smoothing = true }) => {
+    return (
+      <ImageViewProvider value={{ suggestion }}>
+        {(chunkSize ? chunks(regions, chunkSize) : regions).map((chunk, i) => (
+          <RegionsLayer
+            key={`chunk-${i}`}
+            name={`chunk-${i}`}
+            regions={chunk}
+            useLayers={useLayers}
+            showSelected={showSelected}
+            smoothing={smoothing}
+          />
+        ))}
+      </ImageViewProvider>
+    );
+  },
+);
+
 const DrawingRegion = observer(({ item }) => {
   const { drawingRegion } = item;
 
   if (!drawingRegion) return null;
   if (item.multiImage && item.currentImage !== drawingRegion.item_index) return null;
 
-  const Wrapper = drawingRegion && drawingRegion.type === "brushregion" ? Fragment : Layer;
+  const isBrush = ["brushregion"].includes(drawingRegion.type);
```

Comment:
do you expect more region types here later? add a comment about this maybe?

Reply:
no. it's actually redundant. previously i also had `bitmaskregion` here but it's handled differently now